release-team/role-handbooks/test-infra/README.md

```diff
@@ -42,6 +44,167 @@ Note that this section reflects the status of the world today, we are actively l
 Not all the steps need to happen together, some new jobs, like bazel-build/integration/verify will require images to be pushed before they can work properly. 
 
 
+### Configure merge automation for code slush, freeze, and thaw
+
+The code slush, code freeze, and code thaw dates in the release cycle mark points at which merge requirements for PRs in the `master` branch and `release-<current-release-number>` change. The remaining branches are `release-X.X` branches for *previous* releases and are unaffected by the release cycle.
+Code slush and freeze are the two phases of the release cycle with additional merge requirements. Code thaw marks the switch back to the development (normal) phase.
+
+#### Tide
+
+The tool that we use to automate merges is called [Tide](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/tide#tide). Its configuration lives in [`config.yaml`](https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml). Tide identifies PRs that are mergeable using GitHub queries that correspond to the entries in the `queries` field.
+Here is an example of what the query config for `kubernetes/kubernetes` looks like without additional constraints related to the release cycle:
+
+```yaml
+  - repos:
+    - kubernetes/kubernetes
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    missingLabels:
```

Comment:
So, these blocks are easy to copy-paste explicitly, I get that.  But the `missingLabels` block never changes.  Maybe show it explicitly once and then `#...` elsewhere or something.  It makes it harder to visually see what's changing.

Reply:
I just did this for `missingLabels` since that is the only big section that is totally static. Let me know if you want me to comment out any other sections.